src/main/java/com/carrotcreative/cream/loaders/multiple/MultipleLoaderTuple.java

```diff
@@ -4,13 +4,12 @@
 
 public class MultipleLoaderTuple {
 
-    public Serializable mContent;
-    public boolean mFromCache;
+    private final Serializable mContent;
```

Comment:
This is also another instance of where they need to remain public.

It's probably best practice if these actually used getters, as opposed to having these values be public -- but in it's current state there's no way to actually access these variables in the `MultipleLoaderCallback`!